core/bones/relationalBone.py

```diff
@@ -2,7 +2,7 @@
 from viur.core.bones import baseBone
 from viur.core.bones.bone import getSystemInitialized
 from viur.core import utils, db
-from viur.core.errors import ReadFromClientError
+from viur.core.errors import BadRequest
```

Comment:
ReadFromClientError lives now in the baseBone, respective
```suggestion
from viur.core.bones.bone import ReadFromClientError
```